api/apitest/harness.go

```diff
@@ -189,7 +189,7 @@ func (h *Harness) testSpan(tracerFactory func() trace.Tracer) {
 			span.AddEventWithTimestamp(context.Background(), time.Now(), "test event")
 		},
 		"#SetStatus": func(span trace.Span) {
-			span.SetStatus(codes.Internal, "internal")
+			span.SetStatus(codes.Ok, "internal")
```

Comment:
Should this be `codes.Error`?  `codes.Internal` was an error status before.

Reply:
Seems reasonable. This is testing more so if a span status can be set rather than validating the set status, but I think being consistent in semantics like pointed out is a good idea :+1:.